elastic/kibana
# [Controls] timeslider control

PR creates a time slider control that is functionally compatible with existing Maps time slider (with the intent that time slider control will one day replace the Maps time slider).

Key features:
1) Time slider control is not tied to data view or data view field (just like the global time picker)
2) Control updates dashboard container input state with timeslice. Panels can use both timeRange (from global timepicker) and timeslice (from time slider) to display content. Maps embeddable uses both timeRange and timeslice to use client-side masking when all data can be loaded to client. For all other embeddables, timeslice is used instead of timeRange.
3) Control uses observable from dashboard container to play next frame when all panels for the current frame finish loading.

<img width="600" alt="Screen Shot 2022-08-29 at 1 26 38 PM" src="https://user-images.githubusercontent.com/373691/187282240-0dd23a98-c7a8-462f-915c-4c41dacf1389.png">

To test:
1) Create dashboard with lots of different types of embeddables
2) Add timeslider control
3) mess around with control and verify dashboard responds as expected.
